bin/diff_results

```diff
@@ -246,6 +246,12 @@ def diff(before_file, after_file, summary_filename):
                 base_case['detailed_classification']['slowdown'] == sample['detailed_classification']['slowdown'] and \
                 base_case['detailed_classification']['no steady state'] == sample['detailed_classification']['no steady state']:
             summary[DIFF][vm][bench][CLASSIFICATIONS] = SAME
+        # If the CIs do overlap, but the classification has moved from bad
+        # inconsistent to good inconsistent, then we say the result was better.
+        if summary[DIFF][vm][bench][CLASSIFICATIONS] == SAME and \
```

Comment:
Would it be correct to say `if summary[DIFF][vm][bench][CLASSIFICATIONS] != BETTER` here? Would that correctly capture additional cases?

Reply:
OK, thanks.